hs_tools_resource/forms.py

```diff
@@ -63,6 +63,32 @@ class Meta:
 class VersionValidationForm(forms.Form):
     value = forms.CharField(max_length=128)
 
+
+class ToolIconFormHelper(BaseFormHelper):
+    def __init__(self, allow_edit=True, res_short_id=None, element_id=None, element_name=None, *args, **kwargs):
+        # the order in which the model fields are listed for the FieldSet is the order these fields will be displayed
+        field_width = 'form-control input-sm'
+        layout = Layout(
+                Field('icon', css_class=field_width),
+        )
+        kwargs['element_name_label'] = 'Tool Icon URL'
```

Comment:
@hyi Users are allowed to edit the tool icon URL. This specifies the url of an image that the user wants as the icon of their webapp, rather than allowing them to upload an icon file. This image will show up on the landing pages of all apps that the app is associated with under "Relevant Tools." If no URL is specified, the general webapp icon created by Mauriel is displayed instead. Does that answer your question?